composer.json

```diff
@@ -24,7 +24,7 @@
         "doctrine/phpcr-odm": "~1.2"
     },
     "require-dev": {
-        "symfony-cmf/testing": "^1.3",
+        "symfony-cmf/testing": "dev-fix_initializers",
```

Comment:
TODO: revert to the right version once https://github.com/symfony-cmf/Testing/pull/114 has been merged and a version tagged.